services/QuillLMS/app/models/concerns/translatable.rb

```diff
@@ -0,0 +1,92 @@
+# frozen_string_literal: true
+
+# app/models/concerns/translatable.rb
+module Translatable
+  extend ActiveSupport::Concern
+
+  SPANISH_LOCALE = "es-la"
+  DEFAULT_LOCALE = SPANISH_LOCALE
+  GENGO_SOURCE = "gengo"
+  OPEN_AI_SOURCE = "open_ai"
+  SOURCES = [OPEN_AI_SOURCE, GENGO_SOURCE]
+
+  included do
+    has_many :translation_mappings, as: :source
```

Comment:
`:example_filename` and `:translatable_attribute` values don't appear to change across instances within a class so I think `class_attribute` might make more sense here.


I'd also like to suggest a couple of renames: 
:translatable_attribute -> :translatable_field_name.  # distinguishes column name vs the value
:example_filename -> :translatable_examples.    # prepend translatable to help differentiate it from non concern methods

```
included do
  class_attribute :translatable_field_name, default: nil
  class_attribute :translatable_examples, default: nil
  ...
end
```

Then in a class the configuration seems more natural:

```
class Question < ApplicationRecord
  include Translatable

  translatable_field_name :instructions
  translation_examples "questions.json"
  ...
```